Makefile

```diff
@@ -50,3 +50,11 @@ clear-cache:
 	docker rm -vf $$(docker ps -a -q) 2>/dev/null ; \
 	docker rmi -f $$(docker images -a -q) 2>/dev/null ; \
 	docker volume rm $$(docker volume ls -q) 2>/dev/null ; true
+
+stop-trial-runners:
+	docker stop $$(docker ps -f "name=fuzzbench-r-*" -q) 2>/dev/null ; \
+	docker rm -vf $$(docker ps -f "name=fuzzbench-r-*" -q) 2>/dev/null ; true
```

Comment:
Have you considered using a single `docker rm --force` instead of `docker stop` and then `docker rm`?
https://docs.docker.com/engine/reference/commandline/rm/#force-remove-a-running-container

Reply:
Thanks! `docker rm --force` is cleaner and what we really want to achieve.